manual/manual/tutorials/coreexamples.etex

```diff
@@ -626,7 +634,12 @@ The source code must be put in a file with extension ".ml". It
 consists of a sequence of phrases, which will be evaluated at runtime
 in their order of appearance in the source file. Unlike in interactive
 mode, types and values are not printed automatically; the program must
-call printing functions explicitly to produce some output. Here is a
+call printing functions explicitly to produce some output.  The ";;" used
+in the interactive examples is not required in
+source files created for use with the OCaml compiler, but can be helpful
```

Comment:
"OCaml **compilers**" and possibly add ", rather than the toplevel (REPL)"?

Reply:
"compilers", yes.

I think it would be a confusing to use "toplevel" twice in one sentence with different meanings.  The term "REPL" would need to be explained, as well.

```
The ";;" used
in the interactive examples is not required in
source files created for use with OCaml compilers, but can be helpful
to unambiguously mark the end of a top-level expression even when
there are syntax errors.
```